### PR TITLE
Enable servlet forward and include tests

### PR DESCRIPTION
**Pull request by laurit**

### CODE REVIEW

**Review by trask — COMMENTED** [review]
thanks @laurit!

Instead of adding a new endpoint, I think it's simpler to allow customization of the exception class, I wasn't sure whether it would work, by the time I tried, figured I'd send PR to your PR: https://github.com/laurit/opentelemetry-java-instrumentation/pull/1

File: instrumentation/servlet/servlet-3.0/javaagent/src/test/groovy/TestServlet3.groovy [hunk 1]
```diff
@@ -22,7 +25,14 @@ class TestServlet3 {
...
     @Override
     protected void service(HttpServletRequest req, HttpServletResponse resp) {
-      HttpServerTest.ServerEndpoint endpoint = HttpServerTest.ServerEndpoint.forPath(req.servletPath)
+      String servletPath = req.getAttribute(RequestDispatcher.INCLUDE_SERVLET_PATH)
+      if (servletPath == null) {
+        servletPath = req.servletPath
+      }
+      if (servletPath.contains("exception")) {
+        System.err.println(servletPath)
+      }
```
[the remark this answers is not in the document]
Comment (laurit) [reply instrumentation/servlet/servlet-3.0/javaagent/src/test/groovy/TestServlet3.groovy]: thanks

**Review by trask — APPROVED** [review]

**Review by anuraaga — APPROVED** [review]
This seems nice, but I'm a bit confused on why there seem to only be changes in tests and not business logic?

**Comment by laurit** [comment]
@anuraaga because tests were slightly flawed and didn't work as they were

**Pull request merged**